deploy-travis.sh

```diff
@@ -0,0 +1,32 @@
+if [ ! -z "$access_token" ]; then
```

Comment:
does access_token is available only for master ?

Reply:
We can limit it to master, but anyway we have an if condition in the travis script to limit it to master (or other specific branch)